src/Forms/ExportCustomTextFormat.cs

```diff
@@ -20,6 +19,8 @@ public ExportCustomTextFormat(string format)
             comboBoxNewLine.Items.Add(l.DoNotModify);
             comboBoxNewLine.Items.Add("||");
             comboBoxNewLine.Items.Add(" ");
+            comboBoxNewLine.Items.Add("{nl}{tab}");
+            comboBoxNewLine.Items.Add("{cr}{lf}");
 

```

Comment:
The combobox items only serve as a reminder of the existence of these placeholders. I thought it might be better than showing a tooltip. I started with putting all 4 in one item (`{tab}{nl}{lf}{cr}`), but thought dividing them over two items looks slightly better. The `{cr}{lf}` is sort of a 'natural' combination, which left me with either `{tab}{nl}` or `{nl}{tab}` in the other item, which is not a particularly difficult choice. :)  I never considered adding a separate item for each placeholder.

The combobox text is editable, any combination of placeholders and literal characters is possible. Only users can decide which combination makes sense in their context (including a single `{cr}`).

Not many people are familiar with .Net. I doubt very much that  `{environment.newline}` will be clearer than `{nl}`. But, perhaps, `{newline}` might be considered preferable to `{nl}`.